### PR TITLE
[ MOSIP-27859 ] Adding pkt_cr_dtimes and setting the packet date and …

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**
…time from metadata of the packet

### CODE REVIEW

File: registration-processor/pom.xml [hunk 1]
```diff
@@ -107,11 +107,11 @@
...
-		<registration.processor.core.version>1.1.5.5-P5</registration.processor.core.version>
+		<registration.status.service.version>1.1.5.5-P6-SNAPSHOT</registration.status.service.version>
+		<registration.processor.core.version>1.1.5.5-P6-SNAPSHOT</registration.processor.core.version>
 		<packet.manager.version>1.1.5.5</packet.manager.version>
-		<registration.processor.rest.client.version>1.1.5.5-P5</registration.processor.rest.client.version>
-		<packet.info.storage.service.version>1.1.5.5-P5</packet.info.storage.service.version>
+		<registration.processor.rest.client.version>1.1.5.5-P6-SNAPSHOT</registration.processor.rest.client.version>
+		<packet.info.storage.service.version>1.1.5.5-P6-SNAPSHOT</packet.info.storage.service.version>
 		<packet.bio.dedupe.service.impl.version>1.1.5.5</packet.bio.dedupe.service.impl.version>
 		<registration.processor.message.sender.version>1.1.5.5</registration.processor.message.sender.version>
```
Comment (vishwa-vyom) [inline registration-processor/pom.xml]: Why are these and above few versions not changed ?

File: registration-processor/pom.xml [hunk 2]
```diff
@@ -107,11 +107,11 @@
 		<jsch.version>0.1.55</jsch.version>
```
Comment (vishwa-vyom) [inline registration-processor/pom.xml:108]: In this file, i was able to see the below.

<object.store.khazana.version>1.1.5.5-rc2</object.store.khazana.version>

why do we use a release canditate version of dependency in the released version ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/pom.xml (hunk 2 above)
Comment (Khuddusshariff0022) [reply registration-processor/pom.xml:108]: These changes was suggested by you. in this PR.
https://github.com/mosip/registration/pull/1689/files

File: registration-processor/pom.xml (hunk 1 above)
Comment (Khuddusshariff0022) [reply registration-processor/pom.xml]: changed the version from 1.1.5.5 to 1.1.5.5-P6-SNAPSHOT

File: registration-processor/pom.xml (hunk 2 above)
Comment (vishwa-vyom) [reply registration-processor/pom.xml:108]: As discussed, did we confirm from @sowmya695 that all the required changes of 1.1.5.5-rc2 is available in 1.1.5.5 ?

Comment (Khuddusshariff0022) [reply registration-processor/pom.xml:108]: changed khazana version to 1.1.5.5

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**